selfdrive/ui/replay/replay.cc

```diff
@@ -275,7 +275,10 @@ void Replay::startStream(const Segment *cur_segment) {
   it = std::find_if(events.begin(), events.end(), [](auto e) { return e->which == cereal::Event::Which::CAR_PARAMS; });
   if (it != events.end()) {
     car_fingerprint_ = (*it)->event.getCarParams().getCarFingerprint();
-    auto bytes = (*it)->bytes();
+    capnp::MallocMessageBuilder builder;
+    builder.setRoot((*it)->event.getCarParams());
+    auto words = capnp::messageToFlatArray(builder);
```

Comment:
messageToFlatArray returns a temporary object so you need to capture it like this. Before, the first few bytes were junk. @j0akl does this work for you too now?

Reply:
Yes, thanks for the feedback